index.js

```diff
@@ -0,0 +1,7 @@
+var test = 0;
+
+const process = {
```

Comment:
'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).